client/lib/directly/index.js

```diff
@@ -14,45 +20,64 @@ const DEFAULT_RTM_WIDGET_OPTIONS = {
 };
 const DIRECTLY_RTM_SCRIPT_URL = 'http://widgets.wp.com/directly/embed.js';
 const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
-let initializationPromise;
+let _initialize;
+
+/**
+ * Set up global variables and configuration for the Directly library.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S
+ */
+function configureDirectlyGlobals() {
+	// Set up the global DirectlyRTM function, required for the Directly library.
+	// This snippet is pasted from Directly's setup code.
+	window.DirectlyRTM = window.DirectlyRTM || function() {
+		( window.DirectlyRTM.cq = window.DirectlyRTM.cq || [] ).push( arguments );
+	};
+	// Since we can only configure once per pageload, this library only provides a
+	// single global configuration.
+	window.DirectlyRTM( 'config', DEFAULT_RTM_WIDGET_OPTIONS );
+}
 
+/**
+ * Inserts a dummy DOM element that Directly uses to calculate the base URL for its assets.
+ *
+ * Under standard setup, Directly gathers its base URL by inspecting the src attribute of
+ * the <script> used to load the library. Since we've got a custom setup that doesn't include
+ * a <script> tag, we need to fake this by placing a DOM element with id="directlyRTMScript"
+ * and a src with the appropriate base URL.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S
```

Comment:
As far as I know there's no public version of this document I'm able to link to. I'll double-check with out contact there — but if we're freezing our widget version by self-hosting their script, doesn't it also make sense to link to a version-frozen configuration document?